### PR TITLE
Automatic plugin update

### LINKED ISSUE #329: Write GH workflow to update dependecies and

### PR DESCRIPTION
**Pull request by Ukochka**
GH workflow runs` scripts/update-grafana-version.sh` to update all dependencies, plugin.json and docker-compose files for each plugin example, if there are changes it opens a PR

For now it will be only opening PR and if after a week or so it will be working fine, I can add automatic merge

update-grafana-version.sh script runs `create-plugin@latest update`  and after checks the version of grafana/data and sets this as a TARGET_VERSION and updates docker-compose and plugin.json files

fixes #329

### CODE REVIEW

File: .github/workflows/automatic-update.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          branch: new-update-plugin-examples
+          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+          delete-branch: true
+          commit-message: "chore: update configuration to latest version"
+          title: "Chore: Bump plugin-examples configuration to latest version"
+          body: |
+            Bumps [`grafana-plugin-examples`](https://github.com/grafana/grafana-plugin-examples) configuration to the latest version.
+
+            **Notes for reviewer:**
+            This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files and docker-compose files for each plugin example
```
Comment (tolzhabayev) [inline .github/workflows/automatic-update.yml:46]:
```suggestion
            This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files, grafana-plugin-sdk-go and docker-compose files for each plugin example
```

File: .github/workflows/automatic-update.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PEM }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Update Grafana version
+        run: |
+          bash scripts/update-grafana-version.sh
+          git status
```
Comment (tolzhabayev) [inline .github/workflows/automatic-update.yml:25]: do we need `git status` here?

File: .github/workflows/automatic-update.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+name: Update Grafana Version
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```
Comment (tolzhabayev) [inline .github/workflows/automatic-update.yml:5]:
```suggestion
    - cron: "0 8 * * *"
```

might get more attention this way, no need to do it in the night

**Review by academo — APPROVED** [review]
Nice work!

**Review by jackw — APPROVED** [review]
Awesome work @Ukochka LGTM! 🚀

**Pull request merged**